### DISCUSSION TITLE
CSS Issues: Empty square and "Expand/Contract" icon always visible

### DISCUSSION THREAD

**Discussion by 4vgj03** [rpearce/react-medium-image-zoom#598] [Q&A] [answered]
Hi All,

The code is mostly working as expected, but there is always an empty box on the screen, as well as the expand/contract double-arrow symbol.

Here is my code and css 

`import React, { useState, useCallback } from 'react';
import Zoom from 'react-medium-image-zoom';
// import { Controlled as ControlledZoom } from 'react-medium-image-zoom';
import './reactimagestyles.css';
import './Resume.css';
const Resume = () => {

  const [isZoomed, setIsZoomed] = useState(false);
  const handleZoomChange = useCallback(shouldZoom => {
    setIsZoomed(shouldZoom);
  }, []);

  const [searchTerm, setSearchTerm] = useState('');
  const [filteredItems, setFilteredItems] = useState(resumeItems);

  const handleSearch = (e) => {
    const value = e.target.value.toLowerCase();
    setSearchTerm(value);

    const filtered = resumeItems.filter(item => {
      const companyNameText = typeof item.companyName === 'string' ? item.companyName.toLowerCase() : '';
      const descriptionText = typeof item.description === 'string' ? item.description.toLowerCase() : extractTextFromDescription(item.description).toLowerCase();

      return (
        item.title.toLowerCase().includes(value) ||
        companyNameText.includes(value) ||
        descriptionText.includes(value) ||
        item.keywords.some(keyword => keyword.toLowerCase().includes(value)) ||
        isYearInRange(item.years, value)
      );
    });

    setFilteredItems(filtered);
  };

  const extractTextFromDescription = (element) => {
    if (typeof element === 'string') {
      return element;
    }

    if (Array.isArray(element)) {
      return element.map(extractTextFromDescription).join(' ');
    }

    if (element.props.children) {
      return extractTextFromDescription(element.props.children);
    }

    return '';
  };

  const isYearInRange = (years, searchYear) => {
    const yearRange = years.toString().split('-');
    if (yearRange.length === 2) {
      const startYear = parseInt(yearRange[0]);
      const endYear = parseInt(yearRange[1]);
      const searchYearNum = parseInt(searchYear);
      return searchYearNum >= startYear && searchYearNum <= endYear;
    }
    return years.toString().includes(searchYear);
  };

  const renderSection = (title, items) => {
    const filteredItemsInSection = items.filter(item => filteredItems.includes(item));
    if (filteredItemsInSection.length === 0) {
      return null;
    }
  
    return (
      <div className="resume-section">
        <h2 className='h2Resume'>{title}</h2>
        {filteredItemsInSection.map((item, index) => {
          // Filter out items that are not of the specified category
          if (item.category !== title) {
            return null;
          }
          return (
            <div key={index} className="resume-item">
              <img src={item.logo} alt="Logo" className="job-logo" />
              <h3>{item.title}</h3>
              <p className="company">
                <a href={item.companyLink}>{item.companyName}</a> | {item.location}
              </p>
              <p className="years">{item.years}</p>
              <div className="description">{item.description}</div>
              {/* Include Zoom component for certificates */}
              {title === 'Certificates' && item.title === 'The Complete 2024 Web Development Bootcamp' && (
                <Zoom isZoomed={isZoomed} onZoomChange={handleZoomChange}>
                  <img
                    alt="That wanaka tree, alone in the water near mountains"
                    src={WebCert}
                    width="500"
                  />
                </Zoom>
              )}
              {title === 'Certificates' && item.title === 'PMP' && (
                <Zoom isZoomed={isZoomed} onZoomChange={handleZoomChange}>
                  <img
                    alt="PMP Certificate"
                    src={PMPCert}
                    width="500"
                  />
                </Zoom>
              )}
            </div>
          );
        })}
      </div>
    );
  };
  

  const workExperienceItems = resumeItems.filter(item => item.category === 'Work Experience');
  const educationItems = resumeItems.filter(item => item.category === 'Education');
  const certificateItems = resumeItems.filter(item => item.category === 'Certificates');

  return (
    <div className="resume-page">
      <div className="resume-container">
        <input
          type="text"
          className="search-bar"
          placeholder="Search for any word or year..."
          value={searchTerm}
          onChange={handleSearch}
        />
        <div className="resume-list">
          {renderSection('Work Experience', workExperienceItems)}
          {renderSection('Certificates', certificateItems)}
          {renderSection('Education', educationItems)}
        </div>
      </div>
    </div>
  );
};`

CSS FILE
`/* src/components/Resume.css */

body, html {
  height: 100%;
  margin: 0;
}

h2Resume {
  color: #66BFBF;
  text-align: center;
  font: 2.5rem;
  font-family: 'Montserrat', sans-serif;
  font-weight: normal;
  padding-bottom: 10px;
}

.resume-page {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
}

.resume-container {
  padding: 20px;
  max-width: 800px;
  margin: auto;
  margin-bottom: 100px;
  flex: 1; /* Take up remaining space */
}

.search-bar {
  width: 100%;
  padding: 10px;
  margin-bottom: 20px;
  border: 1px solid #ccc;
  border-radius: 4px;
  font-size: 16px;
}

.resume-list {
  display: flex;
  flex-direction: column;
  gap: 30px;
  margin: 10px;
}

.resume-item:hover {
  transform: scale(1.02);
}

.resume-item h3 {
  margin: 0 0 10px 0;
  font-size: 1.5rem;
}

.company {
  font-weight: bold;
}

.years {
  font-style: italic;
}

.description {
  margin-top: 10px;
}

.resume-item {
  position: relative; /* Position relative for absolute positioning of logo */
  padding: 20px;
  border-radius: 4px;
  color: #000;
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  transition: transform 0.2s ease-in-out;
  /* Add relative positioning to the resume item */
  position: relative;
  background-color: rgba(255, 255, 255, 0.6); /* White background with 90% opacity */
  margin-top: 30px;
}

.job-logo {
  position: relative;
  top: -20px; /* Adjust top position as needed */
  left: -20px; /* Adjust left position as needed */
  width: 50px; /* Adjust size as needed */
  height: 50px; /* Adjust size as needed */
  border-radius: 50%; /* Optional: makes the logo circular */
  /* Position the logo relative to its parent (.resume-item) */
  position: absolute;
  z-index: 1;
}

.menu {
  position: relative; /* or absolute/fixed if already set */
  z-index: 10; /* Any number higher than the logo's z-index */
}

.bottom-container {
  text-align: center;
  padding: 20px;
  background-color: #333;
  color: white;
  /* Ensure the footer is not fixed */
  position: static;
}

.footer-link {
  color: white;
  margin: 0 10px;
  text-decoration: none;
}

.footer-link:hover {
  text-decoration: underline;
}

.copyright {
  margin-top: 10px;
}
`
![image](https://github.com/rpearce/react-medium-image-zoom/assets/104655397/0532777a-c948-43fd-ad24-873a00440b4e)

**Comment by rpearce** [comment, marked as answer]
@4vgj03 It looks like you need to import the stylesheet:

```tsx
import 'react-medium-image-zoom/dist/styles.css'
```

Or you can copy-paste the code, yourself, from https://github.com/rpearce/react-medium-image-zoom/blob/main/source/styles.css.
